README.md

```diff
@@ -54,11 +54,16 @@ docker build . --tag=silo
 Run docker container
 
 ```shell
-docker run -p 8081:8081 silo
+docker run -p 8081:8081 -v your/data/directory:the/directory/in/docker/image silo --api --preprocessingConfig=the/directory/in/docker/image/test_preprocessing_config.yaml --databaseConfig=the/directory/in/docker/image/test_database_config.yaml
 ```
 
+The mounted directory `your/data/directory` is the place on your machine, where the data for SILO is located. The data
+will be mounted to the directory `the/directory/in/docker/image` in the docker image. Both the preprocessingConfig and
```

Comment:
Do we really want the user to provide the location in the docker image? Can we not just use `data` here? At least it is not quite clear to me, when someone would actually change the path where the docker stores internal state

Reply:
I had that in there, because we specify in the preprocessing config, where the data lies. But as you say it would be better, to keep that away from the user.

Reply:
I see that the test docker compose file sets it as `/app/testBaseData`, but maybe we stay even more generic, but still under `/app` if that is wanted?

Reply:
But I have no clear preference over `/app/data` or any other descriptive naming